.github/workflows/python-app.yml

```diff
@@ -0,0 +1,27 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.6
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.6
+    - name: Install dependencies
+      run: |
+        pip install --upgrade pip
+        pip install .
+        pip install -r tests-requirements.txt
+    - name: Linter
+      run: |
+        pylama
```

Comment:
Can you also add a type checking task ?

Reply:
I don't know how to do it ...

In the yarn repo I have this : 
    - name: Typer checker
      run: |
        mypy --ignore-missing-imports --config-file setup.cfg

Reply:
Yes, you can put it as is and change the setup.cfg

Reply:
We can maybe add thisalso  as in thx:
mypy . --ignore-missing-imports --disallow-untyped-defs --no-strict-optional